src/locales/de/app.json

```diff
@@ -0,0 +1,132 @@
+{
+	"loading": "Lade...",
+	"download-ready": "Jetzt herunterladen!",
+	"navigation": {
+		"home": "Start",
+		"configuration": "Kalender erzeugen",
+		"features": "Features",
+		"faq": "FAQ"
+	},
+	"language": {
+		"label": "Sprache",
+		"english": "Englisch",
+		"polish": "Polnisch",
+		"french": "Französisch",
+		"spanish": "Spanisch",
+		"german": "Deutsch"
+	},
+	"configuration": {
+		"selector": {
+			"label": "Start",
+			"template": {
+				"label": "Wähle eine Vorlage aus:",
+				"basic": "Einfach",
+				"advanced": "Fortschritten",
+				"blank": "Leer",
+				"minimalistic": "Minimalistisch"
+			},
+			"upload": {
+				"label": "Lade einen existierenden ReCalendar PDF hoch um die Konfiguration zu übernehmen:",
```

Comment:
Suggestion:

"Lade ein existierendes ReCalender PDF hoch um die Konfiguration zu übernehmen:"

Reply:
Yes this is definitely a typo.

I will fix that, instead of "existierenden" I will also word this as "bestehenden" on second thought this seems more appropriate.